src/Symfony/Component/Config/FileLocator.php

```diff
@@ -42,11 +42,19 @@ public function locate($name, $currentPath = null, $first = true)
         }
 
         if ($this->isAbsolutePath($name)) {
-            if (!file_exists($name)) {
+            if (file_exists($name)) {
+                return $name;
+            }
+
+            if (array() === $files = glob($name, defined('GLOB_BRACE') ? GLOB_BRACE : 0)) {
                 throw new FileLocatorFileNotFoundException(sprintf('The file "%s" does not exist.', $name));
             }
 
-            return $name;
+            if (true === $first) {
```

Comment:
This would always have returned only one result, so the check for `$first` wasn't necessary. But the `glob` could return more than one result, so the the check needed to be added